web/views.py

```
@@ -7103,3 +7103,26 @@ def users_list(request: HttpRequest) -> HttpResponse:
     }
 
     return render(request, "users_list.html", context)
+
+#Function free_checkout to handle enrollments without payment
+from django.views.decorators.http import require_POST
+from django.http import JsonResponse
+from .models import Cart, Enrollment
+
+@require_POST
+def free_checkout(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `free_checkout`

(ANN201)